File: TvShowApp/app/src/main/java/com/gm/gislain/tvshowapp/Model/Creator.java
package com.gm.gislain.tvshowapp.Model;

/**
 * Created by Gislain on 25/11/2016.
 */

public class Creator {

    private int id;
    private String creator;

    public static final String KEY_ID = "id";
    public static final String KEY_CREATOR = "creator";

    public Creator() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCreator() {
        return creator;
    }

    public void setCreator(String creator) {
        this.creator = creator;
    }
}

File: TvShowServer/Model/creator.py
class Creator(object):
    def __init__(self, id, creator):
        self.id = id
        self.creator = creator

    def getId(self):
        return self.id

    def getCreator(self):
        return self.creator

    def serialize(self):
        return {'id': self.id, 'creator': self.creator}
